### PR TITLE
Add How to Join, Interested Party and Emeritus sections

### PR DESCRIPTION
**Pull request by samueltauil**
Signed-off-by: Samuel Tauil <samueltauil@gmail.com>

### CODE REVIEW

File: sigs/sig-marketing/README.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ Sig marketing meetings are held twice monthly. Meetings can be found in the [Arg
 | Member | Create and approve official communications (emails, social, blogs, etc). Manage and work on official events. Work in committee on projects as deemed appropriate by ArgoProj maintainers. Provide regular updates to maintainers on marketing initiatives. |
 | Interested Party | Provide support, create content, and assist with sig-marketing initiatives. No nomination is required to be added as an interested party. |
 | Lead | Coordinates work in the SIG. Leads and runs the SIG Marketing meetings. Reports back on SIG Marketing completed and upcoming work to maintainers. |
+| Emeritus | Inactive members that can eventually become active again without the need for additional processes.
```
Comment (christianh814) [inline sigs/sig-marketing/README.md:21]: Would the process be a PR changing the status from Emeritus to Member?

File: sigs/sig-marketing/README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Argo Sig Marketing coordinates efforts between interested parties to promote, ad
 Sig marketing meetings are held twice monthly. Meetings can be found in the [Argo Project public calendar](https://calendar.google.com/calendar/embed?src=argoproj%40gmail.com&ctz=America%2FDenver).
 
 ## Membership Requirements 
-- Members are nominated and approved by Argo Proj maintainers. 
+- Members are nominated and approved by other members and/or maintainers.
```
Comment (christianh814) [inline sigs/sig-marketing/README.md:9]: Should this say

"...other Marketing SIG Members and/or Argo Project Maintainers"

We should still be as specific as we can while maintaining flexibility

**Review by christianh814 — COMMENTED** [review]
Left some thoughts

File: sigs/sig-marketing/README.md (hunk 1 above)
Comment (samueltauil) [reply sigs/sig-marketing/README.md:21]: Made it more clear that to become active from emeritus a PR would be needed to request that move back.

File: sigs/sig-marketing/README.md (hunk 2 above)
Comment (samueltauil) [reply sigs/sig-marketing/README.md:9]: Change made as suggested.

**Review by christianh814 — APPROVED** [review]
This LGTM

File: sigs/sig-marketing/README.md [hunk 3]
```diff
@@ -35,3 +39,11 @@ Sig marketing meetings are held twice monthly. Meetings can be found in the [Arg
 | Jenny Blekh | Octopus Deploy | jenny.blekh@octopus.com |
 | Meghan Murphy | Intuit | Meghan_Murphy@intuit.com |
 | Wojtek Cichoń | Akuity | wojtek@akuity.io  |
+
+## Interested Party
```
Comment (terrytangyuan) [inline sigs/sig-marketing/README.md:43]: How are we planning to use this? This seems like a frequently changing list and requires a lot of maintenance.

File: sigs/sig-marketing/README.md [hunk 4]
```diff
@@ -18,6 +18,10 @@ Sig marketing meetings are held twice monthly. Meetings can be found in the [Arg
 | Member | Create and approve official communications (emails, social, blogs, etc). Manage and work on official events. Work in committee on projects as deemed appropriate by ArgoProj maintainers. Provide regular updates to maintainers on marketing initiatives. |
 | Interested Party | Provide support, create content, and assist with sig-marketing initiatives. No nomination is required to be added as an interested party. |
 | Lead | Coordinates work in the SIG. Leads and runs the SIG Marketing meetings. Reports back on SIG Marketing completed and upcoming work to maintainers. |
+| Emeritus | Inactive members that can eventually become active again (by submitting a PR) without the need for additional processes.
```
Comment (terrytangyuan) [inline sigs/sig-marketing/README.md:21]: Argo Proj member or SIG member? I think process for inactive members is already documented in membership doc.

Comment (samueltauil) [reply sigs/sig-marketing/README.md:21]: This is only applicable to the scope of the SIG marketing.

File: sigs/sig-marketing/README.md (hunk 3 above)
Comment (todaywasawesome) [reply sigs/sig-marketing/README.md:43]: @terrytangyuan This is only for sig-marketing and basically acts as a way for people to document their interest in participating.

**Review by todaywasawesome — APPROVED** [review]
Looks great, thanks for putting this together @samueltauil !

**Review by terrytangyuan — APPROVED** [review]
Thanks!

**Pull request merged**